docs/explanations/architecture/styles.md

```diff
@@ -0,0 +1,143 @@
+## Styles in the block editor
+
+This document introduces the main concepts related to styles in the block editor. It points to the relevant reference guides and tutorials for readers to dig deeper into each one of the ideas presented. It's aimed to block authors and people working in the block editor project.
+
+1. HTML and CSS
+2. User-provided Block Styles
+    - From UI controls to HTML markup
+    - Block Supports API
+    - Current limits of the Block Supports API
+3. User-provided Global Styles
+
+### 1. HTML and CSS
+
+By creating a post in the block editor the user is creating a number of artifacts: a HTML document plus a number of CSS stylesheets, either embedded in the document or external.
+
+On the one hand, the final HTML document is the result of a few things:
```

Comment:
"On the one hand" and "On the other hand" as an expression, made me think of weighing up two options, rather than two parts of a whole. What about:

`The final HTML document markup is the result of a few things:`

and

`The stylesheets loaded in the front end include:`

Reply:
Nice one! https://github.com/WordPress/gutenberg/pull/38208/commits/ac5605868d3bbef9ba44b2cb52ba11a234b6a697